### PR TITLE
[Issue #25] Drug details and edit functionality

### PR DESCRIPTION
**Pull request by kozych**
Added option to view drug details, edit them and delete drug from database

### CODE REVIEW

File: app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt [hunk 1]
```diff
@@ -60,6 +62,13 @@ class FragmentDrugForm : Fragment(), TimePickerDialog.OnTimeSetListener {
         super.onViewCreated(view, savedInstanceState)
         drugActivity = activity as DrugActivity
         setupListeners()
+//        val variant = arguments?.getInt(DetailsFormConstants.FORM_VARIANT_KEY)
+//        if (variant == DetailsFormConstants.FORM_VARIANT_DETAILS ) {
+//            prepareDetailsVariant()
+//        }
+
+        setTextInputsNotEditable()
+        setTextInputsEditable()
```
Comment (kozych) [inline app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt:71]: Line 70 seems to work fine but I still have problem with line 71. I am not able to edit any field in form after executing the function. @michalmobi please help

Comment (michalmobi) [reply app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt:71]: When I run this branch I get an error: "Cause: compileSdkVersion is not specified. Please add it to build.gradle". Did you have any uncommited gradle changes? If so please fix it first so I can run the app

Comment (kozych) [reply app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt:71]: I wasn't changing gradle in this branch at all. I tested this on clean repo and runs fine. Do you have this issue with other branches too?

Comment (kozych) [reply app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt]: Resolving conversation as I already resolved this issue

File: app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt [hunk 2]
```diff
@@ -68,13 +80,102 @@ class FragmentDrugForm : Fragment(), TimePickerDialog.OnTimeSetListener {
...
+            dose.isEnabled = false
+            doseInterval.isEnabled = false
+            doseTime.isEnabled = false
+
+            buttonEdit.setOnClickListener {
+                setEditView()
+            }
+        }
+        printDrugInfo()
+    }
```
[the remark this answers is not in the document]
Comment (kozych) [reply app/src/main/java/com/example/patientassistant/Fragment/FragmentDrugForm.kt]: done

**Review by piotrgos45 — APPROVED** [review]
looks good for me!

**Review by AlicjaKowalska — APPROVED** [review]

**Review by michalmobi — APPROVED** [review]

**Pull request merged**